advanced_functionality/scikit_learn_bring_your_own_LightGBM_model/scikit_learn_bring_your_own_LightGBM_model.ipynb

```diff
@@ -0,0 +1,1121 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    import lightgbm as lgb
</pre><p>Since this notebook focuses on LightGBM, add some introduction and explanation of what this package is, and for which use cases it's recommended.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3084/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>I will do that.</p>

Reply:
Done.